src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+const headerCells = document.querySelectorAll('thead th');
+const secondHeaderCopy = headerCells[1].textContent;
```

Comment:
Ensure that there are at least two header cells before attempting to access `headerCells[1]`. This will prevent potential errors if the table structure is not as expected.